B.READ/B.READ/Sources/Presentation/Library/RecordView/RecordDetailView.swift

```diff
@@ -73,22 +74,40 @@ struct RecordDetailView: View {
       viewModel.send(.onAppear)
       if showAddMenu { UINavigationBar.showOverlay(duration: 0.0) }
     } // : onAppear
+    .onChange(of: needRefresh, { oldValue, newValue in
+      if newValue {
+        viewModel.send(.onAppear)
+        if showAddMenu { UINavigationBar.showOverlay(duration: 0.0) }
+        needRefresh = false
+      }
+    })
```

Comment:
```suggestion
    .onChange(of: needRefresh) { oldValue, newValue in
      if newValue {
        viewModel.send(.onAppear)
        if showAddMenu { UINavigationBar.showOverlay(duration: 0.0) }
        needRefresh = false
      }
    }
```

p3) 그동안 다른 onChange는 이렇게 써서, 통일감 있게 맞춰주시면 좋을 것 같습니다.